src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -152,4 +167,13 @@ public bool MapInboundClaims
     /// Defaults to <see cref="ConfigurationManager{OpenIdConnectConfiguration}.DefaultRefreshInterval" />.
     /// </value>
     public TimeSpan RefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultRefreshInterval;
+
+    /// <summary>
+    /// Gets of sets the <see cref="UseTokenHandlers"/> property to control if <see cref="TokenHandlers"/> or <see cref="SecurityTokenValidators"/> will be used to validate the inbound token.
+    /// The advantage of using the TokenHandlers is:
+    /// <para>There is an Async model.</para>
+    /// <para>The default token handler is a <see cref="JsonWebTokenHandler"/> which is 30 % faster when validating.</para>
+    /// <para>There is an ability to make use of a Last-Known-Good model for metadata that protects applications when metadata is published with errors.</para>
+    /// </summary>
+    public bool UseTokenHandlers { get; set; }
```

Comment:
```suggestion
    public bool UseTokenHandlers { get; set; } = true;
```

+1 for making this the default.

Reply:
Addressed: 507d24137b37c43ddf9c18bdeb7510e70c84bcf1

Reply:
/azp run

Reply:
This appears to have been reverted. Was that intentional?

Reply:
@eerhardt yes, upon reflection, this would be breaking for some users.
@Tratcher would like to see this 'true' by default, but we have time to change this decision if we choose.

Reply:
@brentschmaltz was the default set to true?